modules/appeals_api/app/views/appeals_api/decision_review_mailer/mailer.html.erb

```diff
@@ -42,26 +42,26 @@
   <body>
     <h1><%= @friendly_duration %> <%= @friendly_env %> Decision Review Report from <%= @report.from %> -- <%= @report.to %> </h1>
 
-    <%= render partial: 'appeal_table', locals: { appeal_name: 'HLR', appeal_status_and_count: @report.hlr_by_status_and_count, faulty_appeals: @report.faulty_hlr } %>
+    <%= render partial: 'appeal_table', locals: { appeal_name: 'HLR', appeal_status_and_count: @report.hlr_by_status_and_count, faulty_appeals: @report.faulty_hlr, total_count: @friendly_duration == 'Weekly' ? @report.total_hlr_successes : 0 } %>
```

Comment:
Not really a fan of inline conditionals in the views, but I suppose we can let it slide in this time... Can we just set `total_count` to nil instead of 0 and then we could simplify the logic in the appeal_table partial?

Reply:
Agreed on the inline conditionals :( Updated for nil checks. Thanks!